src/components/DataSession.vue

```diff
@@ -26,6 +27,39 @@ function addOperation(operationDefinition) {
     .catch(error => console.error('Error:', error));
 }
 
+const getImages = async () => {
+  try {
+    const response = await fetch ('http://127.0.0.1:8000/api/datasessions/', {
+      method: 'GET',
+      headers: {
+        'Authorization': 'Token 123456789abcdefg',
+        'Content-Type': 'application/json; charset=UTF-8'
+      }
+    })
+
+    if (!response.ok) {
+      const errorData = await response.json()
+      console.error('Error Response Data:', errorData)
+      throw new Error('Bad request')
+    }
+
+    const data = await response.json()
+    console.log(data)
```

Comment:
Do we need to keep the console.log of data in here right now? Seems useful for debugging but maybe not final code

Reply:
classic me leaving console logs